src/tests/itests/xi_itests.c

```diff
@@ -12,20 +12,21 @@
 #include "xi_itest_tls_layer.h"
 #endif
 #include "xi_itest_mqttlogic_layer.h"
+#include "xi_itest_update.h"
 #undef XI_MOCK_TEST_PREPROCESSOR_RUN
 
 #include "xi_test_utils.h"
 #include "xi_lamp_communication.h"
 
-struct CMGroupTest groups[] = {
-    cmocka_test_group( xi_itests_clean_session ), cmocka_test_group( xi_itests_tls_error )
+struct CMGroupTest groups[] = {cmocka_test_group( xi_itests_clean_session ),
+                               cmocka_test_group( xi_itests_tls_error ),
 #ifndef XI_NO_TLS_LAYER
-                                                      ,
-    cmocka_test_group( xi_itests_tls_layer )
+                               cmocka_test_group( xi_itests_tls_layer ),
 #endif
-        ,
-    cmocka_test_group( xi_itests_mqttlogic_layer ),
-    cmocka_test_group( xi_itests_connect_error ), cmocka_test_group_end};
+                               cmocka_test_group( xi_itests_mqttlogic_layer ),
+                               cmocka_test_group( xi_itests_connect_error ),
+                               cmocka_test_group( xi_itests_update ),
```

Comment:
I would stick with SFT since that's what it's called on the backend.  I know that there's more than just file transfer, including revision tracking, but the feature really is for SFT and that's what we're implementing.